module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -169,6 +169,51 @@ Read the full documentation and download cheatsheets (where available) for these
 
 ### `Seurat` and `SCE` object conversion
 
+This [vignette](https://satijalab.org/seurat/articles/conversion_vignette.html) from the `Seurat` documentation explains how to convert between `SCE` and `Seurat` formats.
+
+In addition, we provide some code examples below for how you can accomplish these conversions.
+For all code examples below, it is assumed that `SingleCellExperiment` and `Seurat` libraries have been loaded in your R environment:
+
+```r
+library(Seurat)
+library(SingleCellExperiment)
+```
+
+
 #### Converting from `Seurat` to `SCE`
 
+The following example code assumes you have a `Seurat` object called `seurat_obj`.
+
+```r
+# Convert Seurat object to SCE object using the
+#  Seurat function `as.SingleCellExperiment()
+sce_object <- as.SingleCellExperiment(seurat_obj)
+```
+
+Alternatively, you can extract individual slots from the `Seurat` object and build your `SCE` object from scratch.
```

Comment:
I feel like introducing this without a reason _why_ we want to do it is not particularly helpful. Is there any case where we would want to extract just one assay with no metadata? If there is a specific case that we feel is worth illustrating, we should do that, but I don't want to encourage people to drop metadata as part of the conversion. 

If there is something that the conversion does not handle automatically, we might want to show how to add that to an existing SCE, or rename things that might end up in unexpected locations, but I'm not sure linking to building an SCE is useful here, especially without the matching docs for how to extract each part of a Seurat object.

Reply:
Hm, maybe this isn't needed at all. I included it as a way to offer some kind of direction if conversion goes wrong, but really if conversion goes _wrong_, a few small details won't help (that's what consultations/office hours are more reasonably for).
Based on the [Seurat source](https://github.com/satijalab/seurat/blob/763259d05991d40721dee99c9919ec6d4491d15e/R/objects.R), it seems like all the bits do get converted, so I'm thinking just call it a day with `Seurat::as.SingleCellExperiment()`.